test/parallel/test-cluster-bind-twice.js

```diff
@@ -18,88 +18,88 @@
 //
 // See https://github.com/joyent/node/issues/2721 for more details.
 
-var common = require('../common');
-var assert = require('assert');
-var cluster = require('cluster');
-var fork = require('child_process').fork;
-var http = require('http');
+const common = require('../common');
+const assert = require('assert');
+const cluster = require('cluster');
+const fork = require('child_process').fork;
+const http = require('http');
 
-var id = process.argv[2];
+const id = process.argv[2];
 
 if (!id) {
-  var a = fork(__filename, ['one']);
-  var b = fork(__filename, ['two']);
+  const a = fork(__filename, ['one']);
+  const b = fork(__filename, ['two']);
 
-  a.on('exit', function(c) {
+  a.on('exit', common.mustCall((c) => {
     if (c) {
       b.send('QUIT');
       throw new Error('A exited with ' + c);
     }
-  });
+  }));
 
-  b.on('exit', function(c) {
+  b.on('exit', common.mustCall((c) => {
     if (c) {
       a.send('QUIT');
       throw new Error('B exited with ' + c);
     }
-  });
+  }));
 
 
-  a.on('message', function(m) {
+  a.on('message', common.mustCall((m) => {
     if (typeof m === 'object') return;
-    assert.equal(m, 'READY');
+    assert.strictEqual(m, 'READY');
     b.send('START');
-  });
+  }));
 
   let ok = false;
 
-  b.on('message', function(m) {
+  b.on('message', common.mustCall((m) => {
     if (typeof m === 'object') return; // ignore system messages
-    assert.equal(m, 'EADDRINUSE');
+    assert.strictEqual(m, 'EADDRINUSE');
     ok = true;
     a.send('QUIT');
     b.send('QUIT');
-  });
+  }));
```

Comment:
No, because that filters out the possibility of system messages... which is a whole other issue. I don't believe we're actually getting any system messages but if we did, then the changes to this test would catch it and flag it as a mustCall count error.